configs/_base_/datasets/semi_partial_coco_detection.py

```diff
@@ -0,0 +1,146 @@
+# dataset settings
+dataset_type = 'CocoDataset'
+data_root = 'data/coco/'
+
+# file_client_args = dict(
+#     backend='petrel',
+#     path_mapping=dict({
+#         './data/': 's3://openmmlab/datasets/detection/',
+#         'data/': 's3://openmmlab/datasets/detection/'
+#     }))
+file_client_args = dict(backend='disk')
+
+color_space = [
+    [dict(type='AutoContrast')],
+    [dict(type='Equalize')],
+    [dict(type='Invert')],
+    [dict(type='Sharpness')],
+    [dict(type='Posterize')],
+    [dict(type='Solarize')],
+    [dict(type='SolarizeAdd')],
+    [dict(type='Color')],
+    [dict(type='Contrast')],
+    [dict(type='Brightness')],
+]
+
+geometric = [[dict(type='Rotate')], [dict(type='ShearX')],
+             [dict(type='ShearY')], [dict(type='TranslateX')],
+             [dict(type='TranslateY')]]
+
+scale = [(1333, 400), (1333, 1200)]
+
+sup_pipeline = [
+    dict(type='LoadImageFromFile', file_client_args=file_client_args),
+    dict(type='LoadAnnotations', with_bbox=True),
+    dict(type='RandomResize', scale=scale, keep_ratio=True),
+    dict(type='RandomFlip', prob=0.5),
+    dict(type='RandAugment', aug_space=color_space, aug_num=1),
+    dict(type='FilterAnnotations', min_gt_bbox_wh=(1e-2, 1e-2)),
+    dict(type='MultiBranch', sup=dict(type='PackDetInputs'))
+]
+
+weak_pipeline = [
```

Comment:
since there are too many pipelines, we should add comments above them to indicate their usages. For example, pipeline used to augment xxx, which will be sent to teacher/student